Lib/cu2qu/__init__.py

```diff
@@ -206,3 +206,48 @@ def curves_to_quadratic(curves, max_errors):
             if s is None or error > max_err:
                 raise ApproxNotFoundError(c, error)
     return splines, errors
+
+
+def elevate_quadratic(p):
+    """Given a quadratic bezier curve, return its degree-elevated cubic."""
+    return [p[0], lerp_pt(p[0], p[1], 2/3), lerp_pt(p[2], p[1], 2/3), p[2]]
+
+
+def merge_curves(p):
+    """ Return the initial cubic bezier curve subdivided in two segments.
+    Input must be a sequence of 7 points, i.e. two consecutive cubic curve
+    segments sharing the middle point.
+    Inspired by an answer on math.stackexchange.com: http://goo.gl/hFFQl0
+    """
+    (x1, y1), (x2, y2), (x3, y3), (x4, y4), (x5, y5), (x6, y6), (x7, y7) = p
+    k = dist((x5, y5), (x4, y4))/dist((x4, y4), (x3, y3))
```

Comment:
in this particular application of the reversed de casteljieu (i.e. converting a non-rational uniform quadratic b-spline to a single cubic bezier), we wouldn't need to take the ration of the distances of the offcurves from the middle oncurve point, since we already know that this will always be interpolated exactly in the middle between them. So, for the first two segments, the ratio would be 1.0, then 0.5, 0.33333, 0.25, etc. as they are added up to each other to form the single curve.
But I also like to keep this merge_curves general, so it can be applied to any two cubic bezier curve segments that were splitted from a single curve at arbitrary `t` value, and not just those that were splitted in order to generate a quadratic b-spline approximation, as it's the case in cu2qu.
Maybe something similar should be part of `fontTools.misc.bezierTools`, possibly generalized to work on curves of any degree, not just cubic...